src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -181,7 +181,7 @@
 		<value>Wstrzymaj wszystko</value>
 	</data>
 	<data name="InProgressTitleTextBlock" xml:space="preserve">
-		<value>Pobierane</value>
+		<value>Pobrane</value>
```

Comment:
Doesn't `Pobrane` mean `Downloaded`?

The 'accurate' translation here is probably `W Trakcie Pobierania` or rather `W Trakcie`, but I imagine that may be long or unfriendly.

Reply:
In the UI, it looks like a "Downloads" folder (a folder that contains downloaded files), just like in Windows Explorer, so the translation in this case is correct.

![Screenshot_20240411-004817~2](https://github.com/Nexus-Mods/NexusMods.App/assets/31535921/c5f55b9e-e993-48db-9120-18c3bf3f15bb)

Reply:
Optionally, I can restore the previous name  `Pobierane`. This is a shorter alternative to `W trakcie pobierania`.

Reply:
Or `Pobieralnia`. Tab where you can download files.

Reply:
`Pobieralnia` sounds great, actually